### DISCUSSION TITLE
Is it possible to use MatchID DIC data for the tracking ?

### DISCUSSION THREAD

**Discussion by NFougeron** [PlusToolkit/PlusLib#1097] [Q&A] [answered]
We use matchID cameras for digital image correlation and we would like to use this setup to track our ultrasound probe and calibration tool for the calibration of the probe. Yet the DIC is performed in post processing and not live in real time. Is it possible to use the PLUS toolkit to calibrate the ultrasound probe in post-processing, with matchID data for example ?

Regards,

**Comment by lassoan** [comment, marked as answer]
You can create a [sequence file](http://perk-software.cs.queensu.ca/plus/doc/nightly/user/FileSequenceFile.html) that contains images and tracking transforms for each image. The header is a very simple text file format and the image data is just the raw pixels.

However, if you need to develop some custom scripts to create the sequence files and DIC cannot even do realtime tracking then you may be better of buying an [OptiTrack Duo](https://www.optitrack.com/cameras/v120-duo/buy.html) tracker for $3k. Plus already supports it, so you don't need to do any custom developments, and it can already do real-time tracking quite accurately.

**Reply by NFougeron** [reply]
Thanks for your reply. So It is possible to do offline calibration? 
I don't need to do it in real time in my case it can be just post processing but I only found the demo videos for real time calibration
